pages/ksphere/kommander/1.1/tutorials/configure-rbac/index.md

```diff
@@ -19,85 +20,97 @@ This procedure requires the following configurations and background:
 Manage access to resources on Kubernetes clusters through the [RBAC API](https://kubernetes.io/docs/reference/access-authn-authz/rbac/). The base premise revolves around assigning users to groups, which are then associated with roles that control their level of access through role bindings.
 This procedure describes how a cluster administrator can assign different levels of access to Kubernetes and Kommander resources in the Kommander UI.
 
-Kommander has [three levels of access control](https://docs.d2iq.com/ksphere/kommander/1.1.0-beta/operations/access-control/) to support global, workspace, and project use cases. Kommander creates default roles at all three levels to simplify the process of assigning users to different levels of access to resources.
+Kommander has [three levels of access control](https://docs.d2iq.com/ksphere/kommander/1.1/operations/access-control/) to support global, workspace, and project use cases. Kommander creates default roles at all three levels to simplify the process of assigning users to different levels of access to resources.
 
 In this procedure, using these default roles, you will create four different personas with users assigned to them:
-- *Global admin*: these users can administer all Kubernetes and Kommander resources on the management cluster and all target clusters.
-- *Global viewer*: these users have read-only viewing access to all Kubernetes and Kommander resources on the management cluster and all target clusters.
-- *Workspace admin*: these users can administer the workspaces they have specifically been given access to.
-- *Workspace viewer*: these users can access the workspaces they have specifically been given access to in a readonly capacity.
 
-If you haven't done so already, create a group to represent each of the personas. For more details on groups, see the [Identity Providers page](https://docs.d2iq.com/ksphere/kommander/1.0/operations/identity-providers/) and the [Identity Provider tutorial](https://docs.d2iq.com/ksphere/kommander/1.1.0-beta/tutorials/authorize-all-users/).
+- _Global admin_: these users can administer all Kubernetes and Kommander resources on the management cluster and all target clusters.
+- _Global viewer_: these users have read-only viewing access to all Kubernetes and Kommander resources on the management cluster and all target clusters.
+- _Workspace admin_: these users can administer the workspaces they have specifically been given access to.
+- _Workspace viewer_: these users can access the workspaces they have specifically been given access to in a readonly capacity.
+
+If you haven't done so already, create a group to represent each of the personas. For more details on groups, see the [Identity Providers page](https://docs.d2iq.com/ksphere/kommander/1.0/operations/identity-providers/) and the [Identity Provider tutorial](https://docs.d2iq.com/ksphere/kommander/1.1/tutorials/authorize-all-users/).
```

Comment:
Since this is a 1.1 page, should we use a 1.1 version of the page `https://docs.d2iq.com/ksphere/kommander/1.0/operations/identity-providers/` ?

Reply:
yeah - but _again_, not changed in this pr 🙈